apps/examples/stdk_smart_switch/device_control.c

```diff
@@ -0,0 +1,176 @@
+/* ***************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <stdio.h>
+#include "device_control.h"
+#include <iotbus/iotbus_gpio.h>
+#include "iot_os_util.h"
+
+static iot_os_queue *button_event_queue = NULL;
```

Comment:
Could you include the header for `NULL`, `OK`?

Reply:
fixed, and also for smart_lamp example, thanks.